README.md

```diff
@@ -1,30 +1,97 @@
+Freeorion builds on the classic *4X* model by incorporating the nation-building
+elements of games such as *Europa Universalis 2*.  While its modular,
+Open Source design allows for a significant degree of customization of the game
+engine and the story elements by the community, the FreeOrion project is
+dedicated to the construction of a living, breathing universe in a
+*grand campaign* model.
 
-## Vision Statement
 
-FreeOrion, an open-source game inspired by Master of Orion, is a turn-based game of epic space strategy that builds on the classic '4X' model by incorporating the nation-building elements of games such as Europa Universalis 2 and a versatile tactical combat engine. While its modular, open-source design allows for a significant degree of customization of the game engine and the story elements by the community, the FreeOrion team is dedicated to the construction of a living, breathing universe in a 'grand campaign' model.
+## Requirements
 
-## How to contribute
+FreeOrion requires an *OpenGL 2.1* capable graphic card and a display with a
+minimum resolution of at least *800x600*.
 
-For more details please see our [Contribution Guidelines](CONTRIBUTING.md).
+FreeOrion requires at least *Windows XP* with *Service Pack 2* or later,
+*Mac OSX 10.7* or later or any reasonable recent Linux distribution on x86
+compatible processors.  Other operating systems and architectures have reported
+to be working by users, but are not actively supported by the FreeOrion project.
+
+
+## Download
+
+[FreeOrion Stable Releases] are the recommend way to play FreeOrion.  Stable
+Releases can be obtained as native installer binaries for Windows and Mac OSX
+or as source releases for Linux and other UNIX-ish platforms from GitHub.
+
+Also some Linux distributions like Fedora, Debian and Arch do provide prebuilt
+packages of FreeOrion.
+
+[FreeOrion Weekly Releases] are in-development releases intended for enthusiasts
+and testers, who want to track or contribute to the development.  Weekly
+Releases can be obtained as native installer binaries for Windows and Mac OSX
+from Sourceforge.
+
+
+## Install
+
+For both Windows and Mac OSX execute the native installer binaries and follow
+the on-screen instructions of the installer to install FreeOrion.
+
+For Linux or other from-source installations in general please refer to the
+[Compile Instructions].
+
+Various Linux distributions provide the stable release of FreeOrion in
+a prebuilt way.  Usually you can install those packages by either using
+a graphical package manager and searching for *FreeOrion* or by installing the
+packages via the command line.
+
+  * [Debian package] stable release: `# apt-get install freeorion`
+  * [Ubuntu package] stable release: `# apt-get install freeorion`
+  * [Fedora package] stable release: `# dnf install freeorion`
+                                  or `# yum install freeorion`
+  * [ArchLinux package] stable release
```

Comment:
Remove the reference to `yum`.  In all released and maintained versions of Fedora `yum` is a link to `dnf` with message about the transition.